publication/tests/test_publication.py

```diff
@@ -135,24 +135,24 @@ def test_distribution_list(self):
         assigned_count = \
             distribution_model.get_product_contract_assigned_count(
                 product_newsletter.id, partner_jan.id)
-        self.assertEqual(assigned_count, 0)
```

Comment:
This should now be set to 40. You got 1 because you forgot to give the quantity value when creating the distribution list entry.